src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -509,19 +513,23 @@ def remove(self,
                interval: Optional[str],
                start_time: Optional[str]) -> None:
         schedule = self.schedules.pop(level_spec.id, None)
+        response = ""
         if schedule:
             if interval is None:
                 schedule = None
             else:
                 try:
-                    schedule.remove(Interval.from_string(interval),
+                    response = schedule.remove(Interval.from_string(interval),
                                     StartTime.from_string(start_time))
                 finally:
                     if schedule:
                         self.schedules[level_spec.id] = schedule
             if not schedule:
                 del self.level_specs[level_spec.id]
+        else:
+            return "Schedule does not exist"
         self.save(level_spec, schedule)
+        return response
```

Comment:
The same here about returning the message this way. And the function signature needs to be updated too.